opendatahub-io/odh-dashboard
# Remove JupyterHub dependencies

Closes #534 

## Description


Remove dependencies related to JupyterHub, most of the YAML docs updates will be addressed in #1636 

## How Has This Been Tested?



1. Make sure you installs the apps and docs in updated manifests/apps/jupyter and make sure your cluster has removed `jupyterhub` OdhApplication CR
2. Set `enabled` to `true` in `notebookController` field in the dashboard config CR
3. Make sure everything works as normal
4. Set `enabled` to `false` in `notebookController` field in the dashboard config CR, and wait for 2 mins
5. Make sure you cannot see any docs, quickstart on the Resources page, Make sure you cannot see the `Jupyter` app card on the Enabled and Explore page
6. Make sure you cannot access the notebook controller via URL
7. Make sure the `Launch Jupyter` button is hidden on the data science projects table
8. Make sure the toleration settings section is hidden in the cluster settings

## Test Impact


N/A

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
